imap-proto/src/core.rs

```diff
@@ -59,6 +59,12 @@ pub fn quoted_data(i: &[u8]) -> IResult<&[u8], &[u8]> {
     Ok((&i[len..], &i[..len]))
 }
 
+/// space = a single space character
+named!(pub space, tag_s!(" "));
```

Comment:
What's the reason to extract this? It doesn't seem to be a separate syntax item in the RFC, and in this case it feels like the abstraction doesn't add much value beyond the 6-character reduction.